Loop/View Controllers/TestingScenariosTableViewController.swift

```diff
@@ -16,7 +16,14 @@ final class TestingScenariosTableViewController: RadioSelectionTableViewControll
 
     private var scenarioURLs: [URL] = [] {
         didSet {
-            options = scenarioURLs.map { $0.deletingPathExtension().lastPathComponent }
+            options = scenarioURLs.map {
+                $0                                                  // /Scenarios/HF-1-Scenario_1.json
+                    .deletingPathExtension()                        // /Scenarios/HF-1-Scenario_1
+                    .lastPathComponent                              // HF-1-Scenario_1
```

Comment:
These (re)naming and organization details seem to be Tidepool specific details as well.  I didn't catch this earlier, but this would also be a responsibility that would make more sense in Tidepool specific code. 

Having the scenario files themselves stored in the TidepoolSupport plugin, and having a SupportUI method like `getScenarios() -> [Scenario]`, with `struct Scenario { var path: String; var name: String}`, or something similar, would keep all the responsibility fairly contained in a Tidepool specific area.

Reply:
Done :)